Update readmes for Image Prioritizer, Embed Optimizer, and Optimization Detective (#1636)
WordPress/performance: plugins/embed-optimizer/readme.txt

== Description ==

> This plugin's purpose is to optimize the performance of [embeds in WordPress](https://wordpress.org/documentation/article/embeds/), such as Tweets, YouTube videos, TikToks, and so on. The current optimizations include:

Reviewer: Maybe have the first sentence on its own, as a short description at a glance.

Also, to me "and so on" sounds too colloquial, maybe better "and others"?

```suggestion
This plugin's purpose is to optimize the performance of [embeds in WordPress](https://wordpress.org/documentation/article/embeds/), such as Tweets, YouTube videos, TikToks, and others.

The current optimizations include:
```
